### PR TITLE
add new S3 static website for aacr interactome

### PR DESCRIPTION
**Pull request by xindiguo**

### CODE REVIEW

File: config/prod/aacr-interactome-s3website-new.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ parameters:
   # The resource owner's email address
   OwnerEmail: "xindi.guo@sagebase.org"
   # Domain name for your website (sagebionetworks.org)
-  DomainName: "sagebase.org"
+  DomainName: "sagebionetworks.org"
   # The sub domain name e.g. ('mysite' in mysite.sagebionetworks.org)
   SubDomainName: "cancer"
   # The Amazon Resource Name (ARN) of an AWS Certificate Manager (ACM) certificate
```
Comment (xindiguo) [inline config/prod/aacr-interactome-s3website-new.yaml]: @zaro0508 @xschildw Do I need a new AcmCertificateArn for this website?

**Comment by xindiguo** [comment]
I was wondering about that. Should I create a PR to remove the file first, then submit another one? The contents are pretty much the same except the domain name

**Comment by zaro0508** [comment]
yes, please create a new PR with new file to provision this new website.  Once you are satisfied with the new site then you can do a new PR to remove the old site.

@xschildw we will also need to to update the travis build to deploy files to the new site as well.

File: config/prod/aacr-interactome-s3website-new.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  # The project for this resource (i.e. NTAP, BSMN, DREAM, etc..)
+  Project: "AACR"
+  # The resource owner's email address
+  OwnerEmail: "xindi.guo@sagebase.org"
+  # Domain name for your website (sagebionetworks.org)
+  DomainName: "sagebionetworks.org"
+  # The sub domain name e.g. ('mysite' in mysite.sagebionetworks.org)
+  SubDomainName: "cancer"
+  # The Amazon Resource Name (ARN) of an AWS Certificate Manager (ACM) certificate
+  AcmCertificateArn: "arn:aws:acm:us-east-1:055273631518:certificate/1fdb2541-963b-4363-963b-69deb71637af"
```
Comment (zaro0508) [inline config/prod/aacr-interactome-s3website-new.yaml]: we have a different cert for sagebionetworks.org.  please replace with `arn:aws:acm:us-east-1:055273631518:certificate/62283392-710d-439e-ad40-3b53a6a78499`

**Comment by xindiguo** [comment]
@xschildw I have just updated the PR with a new file. Please advice on the arn change. Do I need to send an email request to IT? Thanks!

File: config/prod/aacr-interactome-s3website-new.yaml (hunk 2 above)
Comment (xindiguo) [reply config/prod/aacr-interactome-s3website-new.yaml]: Thank you! It has been updated.

**Pull request merged**